tests/BaseApiTestCase.php

```diff
@@ -91,7 +91,7 @@ protected function makeAuthenticatedApiRequest(string $method, string $route, ar
     protected function makeAuthenticatedApiGithubRequest(string $route, array $data)
     {
         $headers = [
-            'X-Hub-Signature' => hash_hmac('sha1', json_encode($data), config('github.secret')),
+            'X-Hub-Signature' => 'sha1=' . hash_hmac('sha1', json_encode($data), config('github.secret')),
```

Comment:
Is this in addition to the changes previously?

Reply:
This is something I deployed to master a while ago, so expected :)